src/common/opt_partition_builder.h

```diff
@@ -151,23 +147,29 @@ class OptPartitionBuilder {
   BufferType GetBufferItem(const std::unordered_map<uint32_t, BufferType>& map_buffer,
```

Comment:
from my perspective renaming with FindBufferItem would be clearer and reflecting implementation logic